xcflushd.gemspec

```diff
@@ -24,19 +24,19 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = '>= 2.1.0'
+  spec.required_ruby_version = '>= 2.3.0'
 
-  spec.add_runtime_dependency "3scale_client", "~> 2.10"
-  spec.add_runtime_dependency "gli", "= 2.14.0"
-  spec.add_runtime_dependency "redis", "= 3.3.2"
-  spec.add_runtime_dependency "hiredis", "= 0.6.1"
-  spec.add_runtime_dependency "concurrent-ruby", "1.0.2"
-  spec.add_runtime_dependency "net-http-persistent", "2.9.4"
-  spec.add_runtime_dependency "daemons", "= 1.2.4"
+  spec.add_runtime_dependency "3scale_client", "~> 2"
```

Comment:
It's literally the last 2.X version. I was seeing whether it would lock to 2.10.Y but seems like it will not and also pick up 2.X, so it's ok to ensure if time went backwards (ie. no 2.11 or 2.10 available) it won't work.